### PR TITLE
Resolve #78 - change reslife to student development

### LINKED ISSUE #78: 1.B Change Residence Life to Student Development
Check one:
- [ ] Semantic Issue: something about the meaning of the text is different
- [x] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:
Change department name to match department we are under

### PR DESCRIPTION
**Pull request by esoccoli**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [X] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Resolve #78

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -76,7 +76,7 @@
 The name of this Special Interest House shall be Computer Science House.
 
 \asection{Derivation of Authority}
-The Computer Science House shall recognize that it receives its right to function as a Special Interest House from the Center for Residence Life.
+The Computer Science House shall recognize that it receives its right to function as a Special Interest House from the Center for Student Development.
```
Comment (rhochgraf21) [inline constitution.tex:79]: Center for Campus Life, perhaps?

**Review by rhochgraf21 — CHANGES_REQUESTED** [review]

**Comment by esoccoli** [comment]
Made requested changes

File: constitution.tex (hunk 1 above)
Comment (danisaba) [reply constitution.tex]: I prefer "Student Development Center" as per RIT links. I do not see any official RIT center for campus life. Robert, why suggest this change?

Check links below:
Student Development Center: https://www.rit.edu/facilitiesmanagement/building/student-development-center
Campus Life: https://www.rit.edu/studentlife/campus-life
Residence Life: https://www.rit.edu/studentlife/residence-life

**Review by danisaba — CHANGES_REQUESTED** [review]
see conversation on this where I linked campus resources per Robert's requested change

**Review by danisaba — APPROVED** [review]

**Review by danisaba — CHANGES_REQUESTED** [review]
We get club rules from the center for campus life and living rules from Reslife. Therefore they have authority and they both need to be mentioned so this needs to be changed

**Review by Sam-Cordry — APPROVED** [review]
Looks good to me now.

**Pull request merged**